modules/azure/aks.go

```diff
@@ -7,7 +7,8 @@ import (
 	"github.com/gruntwork-io/terratest/modules/testing"
 )
 
-// GetManagedClustersClientE is a helper function that will setup an Azure ManagedClusters client on your behalf
+// GetManagedClustersClientE is a helper function that will setup an Azure ManagedClusters client on your behalf.
+// This function would fail the test if there is an error.
```

Comment:
Oh sorry if I wasn't clear, but I meant all the functions that don't have the `E`. The convention is:

- `xxx` function should fail the test instead of returning errors when there is an error, and should include that comment.
- `xxxE` functions should return errors instead of failing the tests.

Reply:
Yes, that makes perfect sense. I missed the "not" and just updated it previously without considering the implication.

Reply:
Made changes, running CI for success and will update when complete.

Reply:
https://github.com/rguthriemsft/terratest/runs/1150630104?check_suite_focus=true

Completed CI successfully.